packages/deployments/contracts/contracts_forge/core/connext/facets/ProposedOwnableFacet.t.sol

```diff
@@ -132,6 +141,45 @@ contract ProposedOwnableFacetTest is ProposedOwnableFacet, FacetHelper {
     utils_acceptNewOwnerAndAssert(_proposed);
   }
 
+  function utils_assignRoleRouter(address routerAgent, address caller) public {
```

Comment:
this isn't a great application for fuzzing imho - I think the only real addresses we need to test for to cover all possible cases are:
- caller == randomAddressA and routerAgent == randomAddressB
- caller == routerAgent == randomAddressA
- caller == 0x0
- routerAgent == 0x0
- caller == routerAgent == 0x0

fuzzing uses a decent amount of resources, so we should pick and choose carefully where we do it

lmk tho if you have some other reasoning here!

Reply:
once we addressZero sanity check, then it should cover all the fuzz cases.